### PR TITLE
feat: incorporate upstream changes

### PR DESCRIPTION
**Pull request by tmatsuo**
* feat: Support MULAW audio encoding
* feat: Support MP3_64_KBPS audio encoding
* feat: Support timepointing via SSML <mark> tag
* feat: Support quota_project_id
* docs: generate and upload docfx yaml files
* testing: add docs-presubmit build

Upstream change:
https://github.com/googleapis/googleapis/commit/a94df49e8f208649f2f5cb39a84668c6a3434ce8

### CODE REVIEW

File: google/cloud/texttospeech_v1/services/text_to_speech/client.py [hunk 1]
```diff
@@ -228,11 +236,11 @@ def list_voices(
...
                 only return voices that can be used to synthesize this
                 language_code. E.g. when specifying "en-NZ", you will
-                get supported "en-\*" voices; when specifying "no", you
-                will get supported "no-\*" (Norwegian) and "nb-*"
+                get supported "en-*" voices; when specifying "no", you
+                will get supported "no-*" (Norwegian) and "nb-*"
                 (Norwegian Bokmal) voices; specifying "zh" will also get
                 supported "cmn-*" voices; specifying "zh-hk" will also
-                get supported "yue-\*" voices.
+                get supported "yue-*" voices.
```
Comment (busunkim96) [inline google/cloud/texttospeech_v1/services/text_to_speech/client.py:243]: I'm not sure why the synth replace isn't working for this, but this is the reason for the docs failure.

Comment (tmatsuo) [reply google/cloud/texttospeech_v1/services/text_to_speech/client.py:243]: Yeah, the autosynth PR also has the same thing:

https://github.com/googleapis/python-texttospeech/pull/70/files#diff-b794d7ad2724dfc5c6821d8d0edb1bebR239

Comment (tmatsuo) [reply google/cloud/texttospeech_v1/services/text_to_speech/client.py:243]: I think I figured it out:

```diff
-    "((en)|(no)|(nb)(cmn)|(yue))-\*",
+    "((en)|(no)|(nb)|(cmn)|(yue))-\*",

```

Comment (tmatsuo) [reply google/cloud/texttospeech_v1/services/text_to_speech/client.py]: Now it's green!

Comment (busunkim96) [reply google/cloud/texttospeech_v1/services/text_to_speech/client.py]: Woo! Thank you! 😁

**Comment by tmatsuo** [comment]
We can merge this, but isn't it better to have autosynth do the job? It'll have a better changelig I think.

**Comment by tmatsuo** [comment]
I'm open to merge this, but in that case, let's have a better commit logs and PR description.

**Comment by tmatsuo** [comment]
@busunkim96 I updated the PR title and description. How does it look?

**Comment by tmatsuo** [comment]
I added `feat: Support quota_project_id` in the description. I think we should put those bullets into the commit log when we squash and merge.

**Pull request merged**